lightning/src/routing/scorer.rs

```diff
@@ -0,0 +1,51 @@
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Utilities for scoring payment channels.
+//!
+//! [`DefaultScorer`] may be given to [`get_route`] to score payment channels during path finding
+//! when a custom [`routing::Score`] implementation is not needed.
+//!
+//! # Example
+//!
+//! TODO: Fill in example once interface has been finalized.
+//!
+//! [`get_route`]: crate::routing::router::get_route
+
+use routing;
+
+/// [`routing::Score`] implementation that provides reasonable default behavior.
+///
+/// Used to apply a fixed penalty to each channel, thus avoiding long paths when shorter paths with
+/// slightly higher fees are available.
+///
+/// See [module-level documentation] for usage.
+///
+/// [module-level documentation]: crate::routing::scorer
+pub struct DefaultScorer {
+	base_penalty_msat: u64,
+}
+
+impl DefaultScorer {
+	/// Creates a new scorer using `base_penalty_msat` as the channel penalty.
+	pub fn new(base_penalty_msat: u64) -> Self {
+		Self { base_penalty_msat }
+	}
+}
+
+impl Default for DefaultScorer {
+	/// Creates a new scorer using 100 msat as the channel penalty.
+	fn default() -> Self {
+		DefaultScorer::new(100)
```

Comment:
nit: I think default should be 500, just kinda gut-feel. Lots of "good" nodes charge 1 sat per payment, and we should probably be willing to prefer that over two-three "free" hops IMO. Maybe even 1k. We can tune this later, but 100 just seems very low.

Reply:
Changed to 500 msats.